packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -147,6 +147,18 @@ const ruleset: any = {
     /// ARM RPC rules for Get patterns
     ///
 
+    // RPC Code: RPC-Get-V1-02
+    GetMustNotHaveRequestBody: {
+      description: "The Get operation must not have a request body.",
+      severity: "error",
+      message: "{{description}}",
+      resolved: true,
+      formats: [oas2],
+      given: "$.paths.*.get.parameters[?(@.in === 'body')]",
```

Comment:
```suggestion
      given: "$[paths,'x-ms-paths'].*.get.parameters[?(@.in === 'body')]",
```

Reply:
whats the difference between these 2?

Reply:
Is it better perf to do it this way?

Reply:
just to cover the paths under 'x-ms-paths'